db/schema.rb

```diff
@@ -11,13 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150609174024) do
+ActiveRecord::Schema.define(version: 20150609191815) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
 
-  create_table "artists", id: :uuid, default: "uuid_generate_v4()", force: :cascade do |t|
+  create_table "artists", id: :uuid, default: nil, force: :cascade do |t|
```

Comment:
Yeah I think enabling the UUID extension again screws up the UUIDs on artist - the default here should be `uuid_generate_v4()` 

http://www.postgresql.org/docs/9.4/static/uuid-ossp.html:

> uuid_generate_v4()	| This function generates a version 4 UUID, which is derived entirely from random numbers.